modules/cluster/addons.tf

```diff
@@ -46,10 +46,12 @@ resource "aws_eks_addon" "kube-proxy" {
 }
 
 resource "aws_eks_addon" "coredns" {
+  count             = var.critical_addons_coredns_enabled == true ? 1 : 0
```

Comment:
I think this is risky actually ... please test, but I think it may force a delete and recreate on an existing cluster!

Reply:
I am guessing we will need some instructions in UPGRADING to explain the manual state migration steps that will be needed!

Reply:
Could I request some support on (or information about) the testing process?

The [docs mention pre-production testing](https://github.com/cookpad/terraform-aws-eks/blob/main/RELEASING.md#branching) - do we have a test cluster configured for this purpose?